lib/avro-patches/schema_validator/schema.rb

```diff
@@ -0,0 +1,9 @@
+Avro::Schema.class_eval do
```

Comment:
Have we checked to make sure we're not clobbering any of our own patches when using `class_eval`?